src/game/Mario.js

```diff
@@ -44,41 +44,11 @@ export const ANIM_TYPE_NO_TRANSLATION = 4
 // translation types the type is set to this
 export const ANIM_TYPE_ROTATION = 5
 
+export const MARIO_ANIM_SLOW_LEDGE_GRAB = 0x00
```

Comment:
going to try to merge this complete list of constant replacements and see how it goes.... but I'm afraid this could end up causing some permanent diverging across other branches with MMO and gameMaster when trying merging with those branches.. but we'll see how it goes.